.github/workflows/reviewdog.yaml

```diff
@@ -29,11 +29,10 @@ jobs:
             sudo apt install shellcheck
         - name: Install bandit
           run: |
-            pip install bandit==1.7.10 && bandit --version
+            pip install --require-hashes --no-deps --no-cache-dir -r .github/workflows/requirements.txt
```

Comment:
do we need both pip install commands? The only difference between line 32 & 35 is the `--no-cache-dir` We can probably combine and rename the stage `Install python requirements` instead of install bandit and pycodestyle